universal_transfer.md

```diff
@@ -0,0 +1,112 @@
+# Universal Transfer Method for Open-Attestation Documents
+
+## Status
+
+Draft
+
+## Rationale
+
+The goal of the universal transfer method is to ensure that a standard is available to all clients interacting with OA documents. This allows different clients to be itneropable with one another. The standard is extremely important if we want to provide a client like an [identity wallet](https://github.com/Open-Attestation/identity-wallet) that supports all kinds of OA document.
+
+Important considerations of the transfer method:
+
+- Reference to resource (uri)
+- Encrypted resource (key)
+- Permitted actions (termsOfUse)
+- Safe transfer (encoding)
+- Universal resolver (deep-linking)
```

Comment:
Do we need to include a TTL to restrict the validity of a QR code?

Reply:
The QR code dont need TTL, the TTL can be returned from the response of the server the file is being uploaded but not required in the QR Code. 

When the QR code is scanned, it either is valid or not by downloading the resource.